client/app/hosting/database/DATABASE_LIST.html

```diff
@@ -150,6 +150,7 @@
                             <td class="word-break"
                                 data-ng-bind="element.name"
                                 data-uib-tooltip="{{element.server}}"
+                                data-tooltip-enable="ctrlTabDatabases.resourceType !== 'cloud'"
```

Comment:
Does [`<oui-tooltip>`](https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/oui-angular/tooltip) component would be suitable here?

Reply:
I did try and use `<oui-tooltip>` but `enable/disable` is not provided by it. Cant be used unless we separate text and tooltip(may be with icon)